components/invitation_modal/invitation_modal_members_step.scss

```diff
@@ -108,23 +108,32 @@
     .invite-members {
         text-align: right;
         margin-bottom: 28px;
-        button {
-            height: 48px;
-            padding: 13px 26px;
-            font-size: 16px;
-            font-weight: 600;
-            width: 180px;
-            border-radius: 4px;
-            background: var(--button-bg);
-            color: var(--button-color);
-            &:focus, &:active, &:focus:active {
-                outline: none;
-                border-color: transparent;
-            }
-            &.btn-inactive {
-                background: var(--center-channel-color-08);
-                color: var(--center-channel-color-56);
-            }
-        }
     }
 }
+
+.InvitationModalMembersStep,
+.FeatureDiscovery {
+    button {
+        height: 48px;
+        padding: 13px 26px;
+        font-size: 16px;
+        font-weight: 600;
+        width: 180px;
+        border-radius: 4px;
+        background: var(--button-bg);
+        color: var(--button-color);
+        &:focus, &:active, &:focus:active {
+            outline: none;
+            border-color: transparent;
+        }
+        &.btn-inactive {
+            background: var(--center-channel-color-08);
+            color: var(--center-channel-color-56);
+        }
+        &.btn-secondary {
+            background: var(--center-channel-bg);
+            border: 1px solid var(--button-bg);
+            color: var(--button-bg);
+        }
+    }
+}
```

Comment:
This was the [recommended approach](https://community-daily.mattermost.com/core/pl/b1ow1n9ee3d79cw5tex33dujno) by the apps team pending the common components.